core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -2890,12 +2891,13 @@ private static class ExtractImplementor extends AbstractRexCallImplementor {
     }
   }
 
-  private static Expression mod(Expression operand, long factor) {
```

Comment:
add javadoc explaining the purpose of the `floorMod` parameter. (If true, negative arguments will yield a negative result.)